e2e/appActions.js

```diff
@@ -480,6 +480,12 @@ async function setEndOffset(page, offset) {
   await setTimeConductorOffset(page, offset);
 }
 
+/**
+ * Set the time conductor bounds in fixed time mode
```

Comment:
I would make a note that we should advocate that the user directly navigates to fixedTime instead of using these appActions after initially navigating to root.

Maybe rename these appActions to something like `switchToTimeConductor` or similar